.github/workflows/smoke-test.yml

```diff
@@ -48,7 +45,9 @@ jobs:
 
       - name: Install
         run: |
-          sudo snap install --dangerous ./docker.snap
+          # dawidd6/action-download-artifact action creates a directory when the name is fetch as a RegExp
+          cd ./docker_*.snap
+          sudo snap install --dangerous ./docker_*.snap
           # it will immediately be failing to start, so let's head that off as quickly as we can
```

Comment:
```suggestion
          sudo snap install --dangerous ./docker_*.snap
          
          # it will immediately be failing to start, so let's head that off as quickly as we can
```